textsubs.lic

```diff
@@ -458,6 +458,13 @@ TextSubs.add('^After a moment, you see a vision of your god who calls to you by
 TextSubs.add('^After a moment, you see a crystal-clear vision of your god who speaks slowly and deliberately, "Your unwavering faith and devotion pleases me greatly, (\w+)\.\s+Go forth and continue your works, and you shall only attain a greater level of purity\."', 'After a moment, you see a crystal-clear vision of your god who speaks slowly and deliberately, "Your unwavering faith and devotion pleases me greatly, \1.  Go forth, and continue your works and you shall only attain a greater level of purity." (15/16)')
 TextSubs.add("^After a moment, you feel a clear presence like a warm blanket covering you beneath the shade of a giant sana'ati tree\.", "After a moment, you feel a clear presence like a warm blanket covering you beneath the shade of a giant Sana'ati tree. (16/16)")
 
+TextSubs.add('You glance about the area while pacing back and forth, like a large, stalking mountain cat.', 'You glance about the area while pacing back and forth, like a large, stalking mountain cat. (extremely wilderness)')
```

Comment:
The `.`s in the "matcher" strings need to be escaped (`\.`).

Reply:
escaping the .'s seems to have fixed it, sorry about that.

You pace back and forth, smiling slightly to yourself at the refreshing familiarity of the area. (moderately wilderness)